bibliopixel/drivers/WS2812SPI.py

```diff
@@ -0,0 +1,50 @@
+from . spi_driver_base import DriverSPIBase, ChannelOrder
+from .. import gamma as _gamma
+
+
+class DriverWS2812SPI(DriverSPIBase):
+    """SPI driver for WS2812(b) based LED strips on devices like the Raspberry Pi, OrangePi, BeagleBone,.."""
+    def __init__(self, num, **kwargs):
+        super(DriverWS2812SPI, self).__init__(num, c_order=ChannelOrder.GRB, SPISpeed=3, gamma=_gamma.WS2812, **kwargs)
```

Comment:
1. Python 3 allows you to simply say `super()`

2. This line might still be too long even with that fix.  `flake8` enforces 79, which is pretty draconian, I agree.

3. Are we sure that all WS2812 strips in future will have that channel order?  You can get exactly the same effect as this code, while still allowing others to override the channel order, like this:

--

     def __init__(self, num, c_order=ChannelOrder.GRB, **kwargs):
         super().__init__(num, c_order=c_order, SPISpeed=3, gamma=_gamma.WS2812, **kwargs)